src/main/java/me/moodcat/database/entities/Room.java

```diff
@@ -45,28 +51,53 @@
     private Integer id;
 
     /**
+     * The name of the room.
+     */
+    @Column(name = "roomName")
+    private String roomName;
```

Comment:
btw this is a duplicate field anyway, slipped in while merging. It should be removed. Name is a bit down :fearful: